clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java

```diff
@@ -829,9 +829,9 @@ public GroupCoordinatorMetrics(Metrics metrics, String metricGrpPrefix) {
             this.heartbeatLatency.add(metrics.metricName("heartbeat-response-time-max",
                 this.metricGrpName,
                 "The max time taken to receive a response to a heartbeat request"), new Max());
-            this.heartbeatLatency.add(metrics.metricName("heartbeat-rate",
-                this.metricGrpName,
-                "The average number of heartbeats per second"), new Rate(new Count()));
+            this.heartbeatLatency.add(new Meter(new Count(),
+                    metrics.metricName("heartbeat-rate", this.metricGrpName, "The average number of heartbeats per second"),
+                    metrics.metricName("heartbeat-total", this.metricGrpName, "The total number of heartbeats")));
```

Comment:
I wonder if we could have a helper method where you passed the metric group name, the name prefix (i.e. `heartbeat`) and the description name ("heartbeats" in this case, but "group joins" in another example) to be used in the description.

Reply:
Done (for the consumer).